events/inventory/findings_base.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Wrong file name, it should be` inventory_base.json`

Reply:
Great catch, just corrected!